convert_trace_annos.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import csv
+from pathlib import Path
+from test2text.db import DbClient
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def trace_test_cases_to_annos(db_path: Path, trace_file_path: Path):
+    db = DbClient(db_path)
+    db.cases_to_annos.init_table()
+
+    test_cases = set()
+    logger.info("Reading trace file and inserting annotations into table...")
+    with open(trace_file_path, mode='r', newline='', encoding='utf-8') as trace_file:
+        reader = csv.DictReader(trace_file)
+        current_tc = ""
+        concat_summary = ""
+        for row in reader:
+            if row.get("TestCase", "") != "":
+                if current_tc != row["TestCase"]:
+                    if current_tc:
+                        case_id = db.test_cases.insert(test_script=row["TestScript"], test_case=row["TestCase"])
+                        annotation_id = db.annotations.insert(summary=concat_summary)
+                        db.cases_to_annos.insert(case_id=case_id, annotation_id=annotation_id)
+                        concat_summary = ""
+                    current_tc = row["TestCase"]
+                    test_cases.add(current_tc)
+                concat_summary += row["Summary"]
```

Comment:
The last testcase is going to be missing. 

add

```py
case_id = db.test_cases.insert(test_script=row["TestScript"], test_case=row["TestCase"])
annotation_id = db.annotations.insert(summary=concat_summary)
db.cases_to_annos.insert(case_id=case_id, annotation_id=annotation_id)
```

also after the iteration

Perhaps, you want to use `itertools.groupby()` for iteration. The code will be easier to read, and you won't need to care about the last group